plugins/modules/sonic_bgp_neighbors.py

```diff
@@ -1064,8 +1069,268 @@
                  prefix_list_out: p2
     state: deleted
 
+# After state:
+# ------------
+#
 # sonic# show running-configuration bgp peer-group vrf default
-# (No bgp peer-group configuration present)
+# !
+# peer-group SPINE
+#  timers connect 30
+#  advertisement-interval 0
+#  !
+#  address-family ipv6 unicast
+#    send-community both
+#  !
+
+#
+# Using replaced
+#
+# Before state:
+# ------------
+#!
+#router bgp 51 vrf VrfCheck2
+# network import-check
+# timers 60 180
+#!
+#router bgp 51 vrf VrfReg1
+# network import-check
+# timers 60 180
+# !
+# peer-group SPINE3
+#  bfd
+#  remote-as 4
```

Comment:
Why are the following default configurations not shown here? (They are shown in the 'after' configuration, making it seem as if the 'replaced' playbook caused them to be there.)

```
 timers connect 30
 advertisement-interval 0
```

Reply:
Updated all examples with correct default configurations in peer-group wherever it is mentioned.

Reply:
Verified.